.drone.yml

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
does this have to be committed in addition to the `.jsonnet` file?

Reply:
Yes, right now it needs to be committed. IMHO it should always be committed and not generated on the fly, as this would make Drone's server a lot more complicated, especially when using external libraries in jsonnet, it would fall apart.

Reply:
Just wanted to understand it.